audit_test.go

```diff
@@ -363,6 +363,14 @@ func TestAuditClientSetBacklogLimit(t *testing.T) {
 		t.Skip("must be root to set rate limit")
 	}
 
+	status, err := getStatus(t)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if status.FeatureBitmap&AuditFeatureBitmapBacklogLimit == 0 {
```

Comment:
Approximately when in Linux was the ability to set the backlog limit added?

Reply:
It was added in 3.14